BoosterKitTests/APIManagerTests.swift

```diff
@@ -6,28 +6,33 @@
 //  Copyright © 2017 Spartan. All rights reserved.
 //
 
-import XCTest
 @testable import BoosterKit
 import Quick
 import Nimble
 import RealmSwift
 import Nocilla
 import KeychainAccess
 
-class APIManagerSpecs: QuickSpec {
+class APIManagerSpecs: BaseSpec {
     override func spec() {
-        LSNocilla.sharedInstance().start()
-        Realm.Configuration.defaultConfiguration.inMemoryIdentifier = self.name
+        beforeEach {
+            // swiftlint:disable:next force_try
+            let realm = try! Realm()
+            // swiftlint:disable:next force_try
+            try! realm.write {
+                realm.deleteAll()
+            }
+        }
```

Comment:
I've never been entirely clear, does this method need a super call too?

Reply:
I don't think so according to the example in the docs: https://github.com/Quick/Quick/blob/master/Documentation/en-us/QuickExamplesAndGroups.md#sharing-setupteardown-code-using-beforeeach-and-aftereach

I have noticed some strange behavior though, so maybe we should look into it a bit more.